accounting_firm/data/mail_alias_domain.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="mail_alias_domain_1" model="mail.alias.domain">
+        <field name="name" model="ir.config_parameter" eval="(obj().get_param('web.base.url') or '').split('//')[-1].split(':')[0]"/>
```

Comment:
Why do you need to do this? Is it a trick with runbot?

Reply:
In alias domain we need to set the name with URL only, without https, so just did it like this.

From
https://82065305-saas-18-2-imported_with_demo.runbot171.odoo.com
To
82065305-saas-18-2-imported_with_demo.runbot171.odoo.com
if base url is set. But yes, in runbot it is taking localhost only, so just removing it.